cypress/integration/smoketests.ts

```diff
@@ -0,0 +1,61 @@
+describe('The Home Page', () => {
+  it('has a working comparator modal', () => {
+    cy.visit('/')
+
+    cy.get('[id^=downshift-0-input]')
+      .click()
+      .type('testing-library/dom-testing-library')
+    cy.wait(5000)
+    cy.contains('p', 'testing-library/dom-testing-library').click()
+
+    // TODO: make selecting first version specific to the first selector box
+
+    cy.contains('select', 'Choose a release').select('v6.16.0')
+    cy.url().should(
+      'eq',
+      'http://localhost:3000/?repo=testing-library%2Fdom-testing-library&from=v6.16.0'
+    )
+
+    // TODO: Select v8.1.0 from second version dropdown box
```

Comment:
I don't know how I can grab the second selector. I could get the first version selector to work though. 🚀 
We're very close to having the full comparator happy path tested. 😉

Reply:
I highly recommend installing [Cypress Testing Library](https://testing-library.com/docs/cypress-testing-library/intro) so you can query elements in an accessible way.

It makes it easier to get the second select by querying by `role: "select"` and `name: "To release"`

Reply:
> It makes it easier to get the second select by querying by `role: "select"` and `name: "To release"`

Yes that would make things way easier! 😄